packages/lesswrong/components/posts/PostsTimeBlock.tsx

```diff
@@ -83,6 +86,16 @@ const PostsTimeBlock = ({ terms, timeBlockLoadComplete, startDate, hideIfEmpty,
     itemsPerPage: 50,
   });
 
+  const filteredPosts = tagFilter ? filter(posts, post => post.tags.map(tag=>tag._id).includes(tagFilter)) : posts
```

Comment:
I don't think we need to import filter from `lodash` if we can just do `posts.filter`